docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1980,32 +2014,48 @@ This section contains reference information about each of the configuration prop
-|`fhirServer/bulkdata/jobParameters/credential.ibm`|string|If use IBM credential, "Y" or "N" |
-|`fhirServer/bulkdata/jobParameters/cos.api.key`|string|API key for accessing IBM COS |
-|`fhirServer/bulkdata/jobParameters/cos.srvinst.id`|string|Service instance Id for accessing IBM COS |
-|`fhirServer/bulkdata/implementation_type`|string|Use "cos" for any S3-compatible object store |
-|`fhirServer/bulkdata/batch-uri`|string|The URL to access the FHIR server hosting the batch web application |
-|`fhirServer/bulkdata/batch-user`|string|User for submitting JavaBatch job |
-|`fhirServer/bulkdata/batch-user-password`|string|Password for above batch user |
-|`fhirServer/bulkdata/batch-truststore`|string|Trust store for JavaBatch job submission |
-|`fhirServer/bulkdata/batch-truststore-password`|string|Password for above trust store |
-|`fhirServer/bulkdata/bulkDataBatchJobIdEncryptionKey`|string|Encryption key for JavaBatch job id |
-|`fhirServer/bulkdata/isExportPublic`|boolean|If give public read only access to the exported files |
-|`fhirServer/bulkdata/validBaseUrls`|string|The list of supported urls which are approved for the fhir server to access|
-|`fhirServer/bulkdata/validBaseUrlsDisabled`|boolean|Disables the URL checking feature|
-|`fhirServer/bulkdata/maxInputPerRequest`|integer|The maximum inputs per bulk import|
-|`fhirServer/bulkdata/cosFileMaxResources`|int|The maximum number of FHIR resources per COS file, "-1" means no limit, the default value is 200000 |
-|`fhirServer/bulkdata/cosFileMaxSize`|int|The maximum COS file size in bytes, "-1" means no limit, the default value is 209715200 (200M) |
-|`fhirServer/bulkdata/patientExportPageSize`|int| The search page size for patient/group export, the default value is 200 |
-|`fhirServer/bulkdata/useFhirServerTrustStore`|boolean| If the COS Client should use the IBM FHIR Server's TrustStore to access S3/IBMCOS service |
-|`fhirServer/bulkdata/enableParquet`|boolean| Whether or not the server is configured to support export to parquet; to properly enable it the administrator must first make spark and stocator available to the fhir-bulkimportexport-webapp (e.g through the shared lib at `wlp/user/shared/resources/lib`) |
-|`fhirServer/bulkdata/ignoreImportOutcomes`|boolean| Control if push OperationOutcomes to COS/S3. |
+|`fhirServer/bulkdata/enabled`| string|Enabling the BulkData operations |
+|`fhirServer/bulkdata/core/api/url`|string|The URL to access the FHIR server hosting the batch web application |
+|`fhirServer/bulkdata/core/api/user`|string|User for submitting JavaBatch job |
+|`fhirServer/bulkdata/core/api/password`|string|Password for above batch user |
+|`fhirServer/bulkdata/core/api/truststore`|string|Trust store for JavaBatch job submission |
+|`fhirServer/bulkdata/core/api/truststorePassword`|string|Password for above trust store |
+|`fhirServer/bulkdata/core/api/trustAll`|boolean|Indicates calls to the local API should skip hostname verification|
```

Comment:
after seeing this written out, I wonder if it should have been named `enableHostnameVerification` (or similar).  in our FHIR Client we use `fhirclient.hostnameVerification.enabled`

Reply:
I say we rock with it for now.  It's in 4.6.0

Reply:
agree